modules/machineset-yaml-alibaba.adoc

```diff
@@ -0,0 +1,176 @@
+    spec:
+      metadata:
+        labels:
+ifndef::infra[]
+          node-role.kubernetes.io/<role>: ""
+endif::infra[]
+ifdef::infra[]
+          node-role.kubernetes.io/infra: ""
+endif::infra[]
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          credentialsSecret:
+            name: alibabacloud-credentials
+          imageId: <image_id> <4>
+          instanceType: <instance_type> <5>
+          kind: AlibabaCloudMachineProviderConfig
+          ramRoleName: <infrastructure_id>-role-<role> <6>
+          regionId: <region> <7>
+          resourceGroup:
+            id: <resource_group_id> <8>
+            type: ID
+          securityGroups:
+          - tags: <9>
+            - Key: kubernetes.io/cluster/<infrastructure_id>
+              Value: owned
+            - Key: GISV
+              Value: ocp
+            - Key: sigs.k8s.io/cloud-provider-alibaba/origin
+              Value: ocp
+            - Key: Name
+              Value: <infrastructure_id>-sg-<role>
+            type: Tags
+          systemDisk: <10>
+            category: cloud_essd
+            size: <disk_size>
+          tag: <9>
+          - Key: kubernetes.io/cluster/<infrastructure_id>
+            Value: owned
+          - Key: GISV
+            Value: ocp
+          - Key: sigs.k8s.io/cloud-provider-alibaba/origin
+            Value: ocp
+          userDataSecret:
+            name: <user_data_secret> <11>
+          vSwitch:
+            tags: <9>
+            - Key: kubernetes.io/cluster/<infrastructure_id>
+              Value: owned
+            - Key: GISV
+              Value: ocp
+            - Key: sigs.k8s.io/cloud-provider-alibaba/origin
+              Value: ocp
+            - Key: Name
+              Value: <infrastructure_id>-vswitch-<region>-<zone>
+            type: Tags
+          vpcId: ""
+          zoneId: <region>-<zone> <12>
+ifdef::infra[]
+        taints: <13>
+        - key: node-role.kubernetes.io/infra
+          effect: NoSchedule
+endif::infra[]
+----
+<1> Specify the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+----
+ifndef::infra[]
+<2> Specify the node label to add.
+<3> Specify the infrastructure ID, node label, and region.
+endif::infra[]
+ifdef::infra[]
+<2> Specify the `<infra>` node label.
+<3> Specify the infrastructure ID, `<infra>` node label, and region.
+endif::infra[]
+<4> Specify the image to use. Note: Get an image from an existing default machine set for the cluster.
+<5> Specify the instance type you want to use for the machine set.
+<6> Specify the name of the RAM role to use for the machine set .
+// Should they use the `ramRoleName` value that the installer populates in the default machine set?
```

Comment:
Question for devs.

Reply:
Yes, it could be the default created by the installer on the default MachineSet. In [AWS doc](https://docs.openshift.com/container-platform/4.9/windows_containers/creating_windows_machinesets/creating-windows-machineset-aws.html#windows-machineset-aws_creating-windows-machineset-aws) we also mention the worker ones:
```
          iamInstanceProfile:
            id: <infrastructure_id>-worker-profile 
```